### ISSUE TITLE
Bug after chapter end

### ISSUE THREAD

**Issue by kiminru** [WinterSquire/AlphaRing#46]
https://youtu.be/E-5kdVB1FAU?si=02GoXN1AKW5WSkKY

(The sound in the YouTube video was not recorded. Please note. 
Although the subtitles are in Korean, you can see the subtitle of the next chapter intro while loading).

The bug occurs after the chapter ends.
To be precise, when a chapter ends, whether it's 2p, 3p, or 4p, the loading part of the next chapter ‘sounds’ like the intro to the next chapter and doesn't progress any further. (It also slows down the loading speed significantly).
Sometimes it works fine, but this bug happens all the time at first.

The build I took this video on is a test build, but I'm currently on 1.0.85 (release) and the same thing is happening. 
(I thought it was only happening on Steamdeck, but it's also happening on my desktop).

**Comment by WinterSquire** [comment]
For the first 'bug', it was mentioned in this [release](https://github.com/WinterSquire/AlphaRing/releases/tag/1.0.85).
I just [fixed](https://github.com/WinterSquire/AlphaRing/commits/master/) the low resolution issue.

**Comment by kiminru** [comment]
> For the first 'bug', it was mentioned in this [release](https://github.com/WinterSquire/AlphaRing/releases/tag/1.0.85). I just [fixed](https://github.com/WinterSquire/AlphaRing/commits/master/) the low resolution issue.

That's great. Can't wait to experience the new release. Oh, and HALO CEA, 2A is the engine used by Saber Interactive, so the graphics in Anniversary for 3P and 4P can't be fixed unless you change the source of the engine, right?

**Comment by WinterSquire** [comment]
Yeah, most of the splitscreen graphics code for h1a and h2a is programmed to 2 players.

**Comment by kiminru** [comment]
> Yeah, most of the splitscreen graphics code for h1a and h2a is programmed to 2 players.

Thanks for the info! :)

**Comment by xCrazyVoidx** [comment]
How did you get no black bars? @kiminru whenever i put in more players its not in fullscreen and theres black bars on each side of the screen

**Comment by kiminru** [comment]
> How did you get no black bars? @kiminru whenever i put in more players its not in fullscreen and theres black bars on each side of the screen

1. are you using the released AlphaRing? Or using a beta AlphaRing?  I'm using the beta AlphaRing.

2. Halo 4 has black bars, the rest of the Halo series does not.

3. If you apply it with 3P from the beginning, you won't see any black bars.

This is the result I got.
